bin/spotxBidAdapter.js

```diff
@@ -49,8 +49,8 @@ export const spec = {
           utils.logError(BIDDER_CODE + ': please define outstream_options parameter or override the default SpotX outstream rendering by defining your own Outstream function using field outstream_function.');
           return false;
         }
-        if (!utils.getBidIdParameter('video_slot', bid.params.outstream_options) || !utils.getBidIdParameter('content_width', bid.params.outstream_options) || !utils.getBidIdParameter('content_height', bid.params.outstream_options)) {
-          utils.logError(BIDDER_CODE + ': please define parameters video_slot, content_width and content_height for object outstream_options in the configuration.');
+        if (!(utils.getBidIdParameter('slot', bid.params.outstream_options) && utils.getBidIdParameter('content_width', bid.params.outstream_options) && utils.getBidIdParameter('content_height', bid.params.outstream_options))) {
```

Comment:
the first && needs to be ||, and the last two conditions need to be in parentheses

Reply:
nope, just one or the other. Easi can figure out the width and height if all we have is a content_container